### PR TITLE
Feat/daily extra 20201013

### PR DESCRIPTION
**Pull request by bfd-kr**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -676,6 +676,11 @@ acorn@^8.2.4, acorn@^8.4.1:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-8.7.0.tgz#90951fde0f8f09df93549481e5fc141445b791cf"
   integrity sha512-V/LGr1APy+PXIwKebEWrkZPwoeoF+w1jiOBUmuxuiUIaOHtob8Qc9BTrYo7VuI5fR8tqsy+buA2WFooR5olqvQ==
 
+acorn@^8.4.1:
+  version "8.7.0"
+  resolved "http://registry.yarnpkg.com/acorn/-/acorn-8.7.0.tgz#90951fde0f8f09df93549481e5fc141445b791cf"
+  integrity sha512-V/LGr1APy+PXIwKebEWrkZPwoeoF+w1jiOBUmuxuiUIaOHtob8Qc9BTrYo7VuI5fR8tqsy+buA2WFooR5olqvQ==
+
```
Comment (andymueller85) [inline yarn.lock]: not really sure why this is getting added. It's already on line 674. i think just undo your yarn.lock changes. When I run yarn it reverts your stuff.

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
   },
   "dependencies": {
     "lodash": "^4.17.20",
+    "ts-node": "^10.7.0",
```
Comment (andymueller85) [inline package.json]: it's already a dev dependency

``
```suggestion
```

**Review by andymueller85 — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (bfd-kr) [reply yarn.lock]: I'm wondering if you made changes since I created my branch

File: package.json (hunk 2 above)
Comment (bfd-kr) [reply package.json]: I had this as a dependency because yarn start was using it.

Comment (andymueller85) [reply package.json]: is starting a dev dependency or an app dependency? seems like an existential question.

**Pull request merged**